site2/docs/sql-getting-started.md

```diff
@@ -21,23 +21,33 @@ To query data in Pulsar with Pulsar SQL, complete the following steps.
 
 ```
 
-2. Start a Pulsar SQL worker.
+2. Adjust configuration.
+
+If you use JDK17 and Pulsar version is less or equal than 2.10, you need to add the following options to the configuration file `conf/presto/jvm.config`.
```

Comment:
In Pulsar 2.10.1 we fixed the issue, see: https://github.com/apache/pulsar/pull/14300
I'm not sure it's even worth to add this document paragraph since it will be deprecated very soon

Reply:
The concrete content and description of this patch may need to adjusted. Considering the users may use old version and also meet the bug like https://github.com/apache/pulsar/issues/15830, I suggest adding this document paragraph even it was solved at new version.